### ISSUE TITLE
metame crash with last radare version

### ISSUE THREAD

**Issue by seifreed** [a0rtega/metame#14]
I updated radare to the last version and metame now crash

```
metame -i ea1da9a6fb202a0e1d9e2c27b1c806b2 -o caca.exe
[INFO] Opening file with r2
Metadata Signature: 0x96e8 0x10001424a5342 12
.NET Version: v2.0.50727
Number of Metadata Streams: 5
DirectoryAddress: 6c Size: 29e0
Stream name: MZ 4
DirectoryAddress: 2a4c Size: 1638
Stream name: MZ 4
DirectoryAddress: 73676e69 Size: 0
Stream name: MZ 4
DirectoryAddress: 10a8 Size: 535523
Stream name: MZ 4
DirectoryAddress: 10 Size: 49554723
Stream name: MZ 4
Traceback (most recent call last):
  File "/usr/local/bin/metame", line 5, in <module>
    metame.main()
  File "/usr/local/lib/python2.7/dist-packages/metame/__init__.py", line 19, in main
    r = r2parser.R2Parser(args.input, True, debug=args.debug, force_replace=args.force)
  File "/usr/local/lib/python2.7/dist-packages/metame/r2parser.py", line 20, in __init__
    info = json.loads(self.r2.cmd("ij").replace("\\", "\\\\"))
  File "/usr/local/lib/python2.7/dist-packages/r2pipe/open_base.py", line 204, in cmd
    res = self._cmd(cmd, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/r2pipe/open_sync.py", line 102, in _cmd_process
    self.process.stdin.write((cmd + '\n').encode('utf8'))
IOError: [Errno 32] Broken pipe
```

Crash with any sample

**Comment by antonioCoco** [comment]
i had the same problem. It seems that "-q" flag breaks the pipe in newer version of radare2.
I managed to fix it with #15 .

**Comment by a0rtega** [comment]
Thanks @antonioCoco for the fix. I merged your PR and released a new version in github and pip.
